torch/testing/_internal/distributed/rpc/jit/rpc_test.py

```diff
@@ -578,6 +582,7 @@ def test_torchscript_functions_not_supported(self):
         # wait for local MyScriptModule instantiation to finish,
         # otherwise it could instantiate MyScriptModule in parallel with
         # server thread in the below
+        initialize_pg(self.init_method, self.rank, self.world_size)
```

Comment:
`dist_init` hasn't initialized the pg already?

Reply:
nvm, just realized that `initialize_pg` guarded that with `if not dist.is_initialized():`, and we would need it for non pg RPC backend.